platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/annotations/Hole.java

```diff
@@ -12,7 +12,7 @@
 /**
  * Encapsulates a {@link List} of {@link LatLng} points defining a hole
  */
-public class Hole extends ArrayList<LatLng> implements Parcelable {
+class Hole extends ArrayList<LatLng> implements Parcelable {
```

Comment:
If the outermost class is parelable it can take care of all it's members. No need for a separate class for the holes in that case. Just move the logic over to Polygon.